tests/system/node/0/config.yaml

```diff
@@ -19,6 +19,8 @@ node:
 
 consensus:
   validator_cycle: 20
+  slash_threshold: 0
+  slash_penalty_amount: 10000
```

Comment:
No need to write it if you're using the default value

Reply:
Right. But It's just a template so that a user can change it.